ext/com_dotnet/com_saproxy.c

```diff
@@ -108,13 +108,18 @@ static zval *saproxy_read_dimension(zval *object, zval *offset, int type, zval *
 		}
 		ZVAL_COPY_VALUE(&args[i-1], offset);
 
-		convert_to_string(&proxy->indices[0]);
+		if (!try_convert_to_string(&proxy->indices[0])) {
+			efree(args);
+			return rv;
+		}
 		VariantInit(&v);
 
 		res = php_com_do_invoke(proxy->obj, Z_STRVAL(proxy->indices[0]),
 			   	Z_STRLEN(proxy->indices[0]), DISPATCH_METHOD|DISPATCH_PROPERTYGET, &v,
 			   	proxy->dimensions, args, 0);
 
+		efree(args);
```

Comment:
This was leaking before?

Reply:
Yes, but it is only reported for master (ZendMM and MSVC Debug Heap), but not for the other branches, although I'm pretty sure that it's leaking there as well. Might be better to fix this in a separate commit for PHP-7.3+.

Reply:
Done, and rebased.